lib/tilex/notifications/notifiers/twitter.ex

```diff
@@ -16,7 +18,40 @@ defmodule Tilex.Notifications.Notifiers.Twitter do
     :ok
   end
 
-  def send_tweet(text) do
-    ExTwitter.update(text)
+  def send_tweet(message) do
+    url = "https://api.x.com/2/tweets"
```

Comment:
could we hoist this up to something like this -
```
@tweet_url "https://api.x.com/2/tweets"
```

Reply:
yes!